src/Engine/ProtoCore/RuntimeCore.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Linq;
+using ProtoCore.AssociativeGraph;
+using ProtoCore.AssociativeEngine;
+using ProtoCore.AST;
+using ProtoCore.AST.AssociativeAST;
+using ProtoCore.BuildData;
+using ProtoCore.CodeModel;
+using ProtoCore.DebugServices;
+using ProtoCore.DSASM;
+using ProtoCore.Lang;
+using ProtoCore.Lang.Replication;
+using ProtoCore.Runtime;
+using ProtoCore.Utils;
+using ProtoFFI;
+
+using StackFrame = ProtoCore.DSASM.StackFrame;
+
+namespace ProtoCore
+{
+    /// <summary>
+    /// RuntimeCore is an object that is instantiated once across the lifecycle of the runtime
+    /// This is the entry point of the runtime VM and its input is a DS Executable format. 
+    /// There will only be one instance of RuntimeCore regardless of how many times instances of a DSASM.Executive (runtime VM) is instantiated.
+    /// Its properties will be persistent and accessible across all instances of a DSASM.Executive
+    /// </summary>
+    public class RuntimeCore
+    {
+        public RuntimeCore(Options runtimeOptions, Executable executable, ProtoCore.Runtime.Context context)
+        {
+            this.context = context;
+            this.executable = executable;
+            this.runtimeOptions = runtimeOptions;
+        }
+
+        public StackValue Bounce(
```

Comment:
Yea, I'm trying to see if this can be a method of executive that bounces to another executive. 
such as:
svReturn = exec.Bounce(other_exec)

Or somthing similar